src/ssh.h

```diff
@@ -0,0 +1,17 @@
+/*
+    SPDX-FileCopyrightText: Lieven Hey <lieven.hey@kdab.com>
+    SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+
+    SPDX-License-Identifier: GPL-2.0-or-later
+*/
+
+#pragma once
+
+class QString;
+class QStringList;
+class QProcessEnvironment;
+
+QStringList assembleSSHArguments(const QString& deviceName, const QStringList& command);
+QProcessEnvironment sshEnvironment();
```

Comment:
important to note here: how does this differ from the normal process environment?